packages/features/webhooks/components/WebhookTestDisclosure.tsx

```diff
@@ -34,12 +30,24 @@ export default function WebhookTestDisclosure() {
           {t("ping_test")}
         </Button>
       </div>
-      <div className="p-4">
-        {!mutation.data && <em>{t("no_data_yet")}</em>}
-        {mutation.status === "success" && (
-          <div className="overflow-x-auto  text-gray-900">{JSON.stringify(mutation.data, null, 4)}</div>
-        )}
+      <div className="space-y-0 rounded-md border-0 border-neutral-200 bg-white sm:mx-0 md:border">
+        <div className="flex justify-between border-b p-4">
+          <div className="flex items-center space-x-1">
+            <h3 className="font-sm self-center font-medium text-black">{t("webhook_response")}</h3>
+            {mutation.data && (
+              <Badge variant={mutation.data.ok ? "green" : "red"}>
+                {mutation.data.ok ? t("passed") : t("failed")}
+              </Badge>
+            )}
+          </div>
+        </div>
+        <div className="rounded-b-md bg-black p-4 text-xs text-white">
```

Comment:
Tagging @Jaibles - but looking at this it looks good, only thing I can spot is that the font used for "No data yet." should be the `SF Mono` font. This is just the Mac OSX default so you can add `font-mono` to achieve this.

Reply:
Added @emrysal